services/src/main/java/org/exoplatform/news/service/impl/NewsTargetingServiceImpl.java

```diff
@@ -210,7 +210,7 @@ private NewsTargetingEntity toEntity(Metadata metadata) {
         NewsTargetingPermissionsEntity permissionEntity = new NewsTargetingPermissionsEntity();
         if (permission.contains("space")) {
           Space space = spaceService.getSpaceByPrettyName(List.of(permission.split(":")).get(1));
-          permissionEntity.setId(space.getId());
+          permissionEntity.setId("space:" + space.getDisplayName());
```

Comment:
I think that this should be done in the previous PRs

Reply:
> I think that this should be done in the previous PRs

Done in the previous PRs !!